analytics/event.go

```diff
@@ -8,46 +8,28 @@ import (
 	"github.com/gofrs/uuid"
 )
 
-// EventBuilder ...
-type EventBuilder interface {
-	AddProperty(key string, value interface{}) EventBuilder
-	Build() Event
-}
-
-type eventBuilder struct {
-	eventName  string
-	properties map[string]interface{}
-}
-
-// NewEventBuilder ...
-func NewEventBuilder(name string) EventBuilder {
-	return eventBuilder{eventName: name, properties: map[string]interface{}{}}
-}
-
-// AddProperty ...
-func (e eventBuilder) AddProperty(key string, value interface{}) EventBuilder {
-	e.properties[key] = value
-	return e
-}
-
-// Build ...
-func (e eventBuilder) Build() Event {
-	return newEvent(e.eventName, e.properties)
+// EventDTO ...
+type EventDTO struct {
```

Comment:
DTO as in Data Transfer Object? Is this Go lingo for data classes?

Reply:
It's Java lingo for serializable classes 😅